pages/common/az-storage-blob.md

```diff
@@ -0,0 +1,25 @@
+# az storage blob
+
+> Manage blob storage containers and objects in Azure.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/storage/blob>.
+
+- Download a blob to a file path:
+
+`az storage blob download --account-name {{storage_account_name}} --account-key {{storage_account_key}} -c {{container_name}} -n {{path/to/blob}} -f {{./local_name}}`
+
+- Download blobs from a blob container recursively:
+
+`az storage blob download-batch --account-name {{storage_account_name}} --account-key {{storage_account_key}} -s {{container_name}} -d {{ path/to/remote}} --pattern {{filename_regex}}`
```

Comment:
Will the blobs be downloaded to the working directory?
```suggestion
`az storage blob download-batch --account-name {{storage_account_name}} --account-key {{storage_account_key}} -s {{container_name}} -d {{path/to/remote}} --pattern {{filename_regex}}`
```

Reply:
Added destination flag.